src/lsst/cm/tools/core/error_code_decisions.yaml

```diff
@@ -158,11 +186,29 @@ pandaErrorCode:
             rescue: 0
             panda: 0
             intensity: 0.001
+        p_all_pixels_masked:
+            diagMessage: "All pixels masked. Cannot estimate background"
+            function: detection
+            known: 1
+            ticket: ["DM-37837", "DM-37570"]
+            resolved: 1
+            rescue: 0
+            panda: 0
+            intensity: 0.001
+        p_too_many_indicies:
+            diagMessage: "IndexError: too many indices for array: array is 1-dimensional, but 2 were indexed"
+            function: detection
+            known: 1
+            ticket: ["DM-37837", "DM-37570"]
+            resolved: 1
```

Comment:
In this case I've marked `resolved` because the bug identified is marked as fixed. So it should mean that it's more concerning to see the error again. 

Maybe I should come up with a better word for this :)